content/community.md

```diff
@@ -18,11 +18,32 @@ Once you have received your invite, please do the following:
 * Read our [code of conduct]({{< relref "code_of_conduct.md" >}})
 * Enter your full name (this community will work best if we are all real humans), but feel free to add a custom "display name"
 * Add a headline in the "what I do" box (can be "title @ company" or anything that helps people know who they are chatting with)
+  * Important: if you are a software vendor, it is _required_ that you list your affiliation in your profile, at a minimum. If you are often discussing your product or your competitors' products, it's even better if you can highlight your affiliation in your display name.
 * Add a profile photo
 * Introduce yourself in the [#introductions](https://locallyoptimistic.slack.com/messages/CBB4QCX4N) channel
 
 In Slack, remember that channels are free. If you want to start a conversation on a topic that doesn't have a channel dedicated to it, you can create a channel and invite others to join you.
 
+###  Participating in the Slack community
+
+In general, in this community, we trust that everyone will behave in ways that they believe are appropriate. There are some hard-and-fast rules in our [code of conduct]({{< relref "code_of_conduct.md" >}}), which we expect everyone to adhere to: specifically, rules prohibiting harassment and intolerant behavior. However, in general, we prefer to have guidelines for behavior rather than hard-and-fast rules in a way that promotes organic growth and discussion. 
+
+We have had some great discussion in the #communityrules channel about how community members who also work for vendors should behave with respect to their and their competitor's competitors' products. There are three concerns we feel the need to address explicitly:
+
+* It should be clear when someone is making a comment if they are potentially biased
+* We do not want vendors bad-mouthing each others other's products
+* We do not want this slack to become a sales-channel for vendors where conversations are consistently dominated by people promoting their own products.
```

Comment:
Just a thought - do we want to change the words from "We" to "Our community"? Gives it a more inclusive, less "up from on high" feels. Of course, "We" could mean "Our community"...